backend/tests/test_routers.py

```diff
@@ -0,0 +1,26 @@
+import uuid
+
+import httpx
+from dateutil import parser
+from fastapi.testclient import TestClient
+
+from src.backend.constants import API_VERSION_PREFIX
+from src.backend.entrypoint import app
+
+client = TestClient(app)
+
+
+def test_create_user():
+    response = client.post(f"{API_VERSION_PREFIX}/user")
+    assert response.status_code == httpx.codes.CREATED
+    json_result = response.json()
+
+    try:
+        uuid.UUID(json_result.get("id"))
```

Comment:
I think you can just `assert uuid.UUID(json_result.get("id"))` and `assert parser.parse(json_result.get("created_on"))`

We don't need error management in unit tests. You'd be raising the bar too high while completion matters most here 😉